recipes/libcublasmp/meta.yaml

```diff
@@ -0,0 +1,110 @@
+{% set version = "0.2.1.427" %}
+{% set lib_version = version.split(".")[0]+"."+version.split(".")[1]+"."+version.split(".")[2] %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-sbsa" %}    # [aarch64]
+{% set extension = "tar.xz" %}
+
+package:
+  name: libcublasmp
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cublasmp/redist/libcublasmp/{{ platform }}/libcublasmp-{{ platform }}-{{ version }}_cuda{{ cuda_version }}-archive.{{ extension }}
+  sha256: 891e86a28cfe08772f2485e4db7ec8d28292e6a1b0e78967d08404c8371925b5  # [linux64 and cuda_version == 11]
+  sha256: d86648a6e1f1d35eb25af258e1828e483aa07c568b95941f349fc68316f56290  # [linux64 and cuda_version == 12]
+  sha256: 5f5813814343e83eeb743ddb4b1df0b346cdbf2b2026b32a0bc4a349361ec8e4  # [aarch64 and cuda_version == 11]
+  sha256: d6cacf073bf87987b0f20a7a5a168ec503264d295e3aebf9df177326e8fcb286  # [aarch64 and cuda_version == 12]
+
+build:
+  number: 0
+  skip: true  # [not (linux64 or aarch64)]
+  script:
+    - cp -rv include $PREFIX
+    - cp -rv lib $PREFIX
+
+test:
+  commands:
+    - test -L $PREFIX/lib/libcublasmp.so.0
+    - test -f $PREFIX/lib/libcublasmp.so.{{ lib_version }}
+
+outputs:
+  - name: libcublasmp
+    files:
+      - lib/libcublasmp.so.*
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - {{ stdlib("c") }}
+      host:
+        - cuda-version {{ cuda_version }}
+        - libcublas    # [cuda_version == 12]
+        - cudatoolkit  # [cuda_version == 11]
+        - libcal
```

Comment:
Q: I assume we need to version-lock `libcal` here? Was cuBLASMp built against a particular libcal version?

Reply:
The versioning between the two components is different, so I would think there is not a strict version lock? https://developer.download.nvidia.com/compute/cublasmp/redist/redistrib_0.2.1.json

Reply:
The versioning between them is different, and I'm not sure if there is a way to easily say which versions of libcal are compatible with a given version of cuBLASMp.

Reply:
> which versions of libcal are compatible with a given version of cuBLASMp

This is exactly why I was asking because conda recipes allow us to express this version information.

When listed in the host section, it means the package will be (or was!) used to build the current package. The purpose pinning a package in host is 3-fold:
1. To indicate what versions of packages are needed at build time
2. To inform conda-build what versions of packages should be added to the run section (if the included packages have `run_exports`)
3. To allow conda-build to perform linking health checks at the end of the build stage

Since we're not building/compiling libcublasmp in the public, we add `libcal` here to meet the 3rd purpose. @mrogowski 
- what was the `libcal` version used to build cuBLASMp 0.2.1? Was it the bundled version (0.4.3)? 
- does cuBLASMp 0.2.1 need to run against exactly libcal 0.4.3, or any 0.4.x (or any 0.x) would work?

Reply:
We discussed this internally, and while cuBLASMp 0.2.1 should work with any 0.4.x version, we believe at this stage of the development (alpha releases) it is the safest to pin exact versions, i.e., use the latest libcal version available at the time of building cuBLASMp. For cuBLASMp 0.2.1, this would be libcal 0.4.3.

Reply:
```suggestion
        - libcal 0.4.3.*
```